src/pages/UpdateForm/UpdateForm.js

```diff
@@ -170,8 +170,7 @@ class UpdateForm extends Component {
    * @return string
    */
   getTimestamp() {
-    const getTimeStamp = Math.round((new Date()).getTime() / 1000);
-    this.timestamp = new Date(getTimeStamp * 1000);
+    this.timestamp = Math.round((new Date()).getTime() / 1000);
```

Comment:
Same output, but why are we getting the unix timestamp here this way and in GiftaidForm.js via Date.now() ?

Reply:
I don't understand your question.

Reply:
`(new Date()).getTime()` and `Date.now()` practically do the same thing but one is cleaner and more precise.

Reply:
Should we then use the cleaner, more precise one in both cases?

Reply:
Yes we can. `Date.now()` performs better. `new Date().getTime()` is useful if you want a frozen `Date` instance or to convert a `YYYY-MM-DD` to a timestamp.